packages/scandipwa/src/component/ProductCompare/ProductCompare.component.js

```diff
@@ -44,6 +46,30 @@ export class ProductCompare extends Component {
         return products !== nextProducts || isLoading !== nextIsLoading;
     }
 
+    renderScroll() {
+        const { handleScroll } = this.props;
+
+        return (
+            <div block="ProductCompare" elem="Scroller">
+                <div block="ProductCompare" elem="ScrollerInner">
+                    <div
+                      id="scrollerScroll"
+                      block="ProductCompare"
+                      elem="ScrollerScroll"
+                      // eslint-disable-next-line react/jsx-no-bind
+                      onScroll={ () => handleScroll() }
```

Comment:
this anonymous function will be created on each render. what I can suggest is using a reference to the function here, so it will be `<div onScroll={ handleScroll } />`